lib/commons/text/accessible-text-virtual.js

```diff
@@ -104,11 +104,11 @@ function shouldIgnoreHidden(virtualNode, context) {
  * @return {Boolean}
  */
 function shouldIgnoreIconLigature(virtualNode, context) {
-  const { ignoreIconLigature, pixelThreshold, occuranceThreshold } = context;
+  const { ignoreIconLigature, pixelThreshold, occurrenceThreshold } = context;
```

Comment:
This should avoid a break in the API.

```suggestion
  const { ignoreIconLigature, pixelThreshold } = context;
  const occurrenceThreshold = context.occurrenceThreshold ?? context.occurenceThreshold
```

Reply:
Thanks for this! I think you mean "occuranceThreshold" since that was the only version of the typo used in code. "Occurence" was only used in a comment. Added this modified fix in [22d9b95a1d851fae3c850f0a5f125a4ba8bf0438](https://github.com/dequelabs/axe-core/pull/3782/commits/22d9b95a1d851fae3c850f0a5f125a4ba8bf0438).